docs/hello_nextflow/05_hello_operators.md

```diff
@@ -689,7 +695,7 @@ _After:_
 process GATK_JOINTGENOTYPING {
 ```
 
-Remember to keep your process names as descriptive as possible, to maximize readability for your colleagues —and your future self!
+Remember to keep your process names as descriptive as possible, to maximize readability for your colleagues — and your future self!
```

Comment:
this is actually a legal use of em-dash without a space before the word 😄

Reply:
Ooooops, thanks for letting me know. Reverting in 3, 2, 1... 😆